lib/rubocop/cop/autocorrect_logic.rb

```diff
@@ -21,6 +21,38 @@ def autocorrect_enabled?
         return true unless cop_config
         cop_config['AutoCorrect'] != false
       end
+
+      def autocorrect(node, location)
+        # TODO: check the disable_uncorrectable option here
+        disable_uncorrectable = true
+
+        if disable_uncorrectable
+          # default implementation: add `# rubocop:disable #{cop_name}` to the
+          # line before and `# rubocop:enable #{cop_name}` to the line after
+          lambda do |corrector|
+            begin_row, begin_col = *location.source_buffer.decompose_position(location.begin_pos)
+            end_row, end_col = *location.source_buffer.decompose_position(location.end_pos)
+            end_line_length = location.source_buffer.source_line(end_row).length
+
+            # get a range from beginning of the first line to just after the last line
+            # (including the newline)
+            range_by_lines = Parser::Source::Range.new(
+              location.source_buffer,
+              location.begin_pos - begin_col,
+              location.end_pos + (end_line_length - end_col + 1)
+            )
+
+            # get first line's indentation (to use in the comments)
+            begin_line = location.source_buffer.source_line(begin_row)
+            leading_whitespace = begin_line[/^\s*/]
+
+            corrector.insert_before(range_by_lines, "#{leading_whitespace}# rubocop:disable #{name}\n")
```

Comment:
Actually @rrosenblum after spelunking through the codebase a bit more I learned that a `# rubocop:disable` works for any offense--for multiline offenses it just has to occur at the end of the _first_ line.

So maybe _all_ disables from this feature should be single-line disables?